builder/caddy-builder.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+plugins=$@
+
+mkdir -p /src/custom-caddy/cmd/caddy
+cd /src/custom-caddy/cmd/caddy
+
+cat >> main.go <<EOF
+package main
+
+import (
+	caddycmd "github.com/caddyserver/caddy/v2/cmd"
+	_ "github.com/caddyserver/caddy/v2/modules/standard"
+EOF
+
+for p in $plugins; do
+    pkg=$(echo $p | cut -f1 -d@)
+    echo -e "\t_ \"$pkg\"" >> main.go
+done
+
+cat >> main.go <<EOF
+)
+
+func main() {
+	caddycmd.Main()
+}
+EOF
+
+cat >> go.mod <<EOF
+module caddy
+
+go 1.14
```

Comment:
Are you sure this can't be templated/env'd? Just thinking for maintenance, might be prone to error to forget to update this line if the required go version changes

Reply:
@francislavoie hrm... actually there's already a `GOLANG_VERSION` env var given by the base image. So that's not too complicated.

Reply:
Solved - PTAL!